stock_available/views/product_template_view.xml

```diff
@@ -0,0 +1,92 @@
+                >
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value d-flex gap-1">
+                            <field
+                                name="immediately_usable_qty"
+                                widget="statinfo"
+                                nolabel="1"
+                                class="oe_inline"
+                            />
+                            <field name="uom_name" class="oe_inline" />
+                        </span>
+                        <span class="o_stat_text">Available</span>
+                    </div>
+                </button>
+                <button
+                    type="object"
+                    name="action_open_quants"
+                    invisible="not is_storable"
+                    class="oe_stat_button"
+                    icon="fa-building-o"
+                >
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value d-flex gap-1">
+                            <field
+                                name="potential_qty"
+                                widget="statinfo"
+                                nolabel="1"
+                                class="oe_inline"
+                            />
+                            <field name="uom_name" class="oe_inline" />
+                        </span>
+                        <span class="o_stat_text">Potential</span>
+                    </div>
+                </button>
+            </xpath>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_stock_available_product_template_list">
+        <field name="name">Quantity available to promise (list)</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.view_stock_product_template_tree" />
+        <field name="arch" type="xml">
+            <field name="virtual_available" position="after">
+                <field
+                    name="immediately_usable_qty"
+                    decoration-danger="immediately_usable_qty &lt; 0"
+                    optional="show"
+                    groups="stock.group_stock_user"
+                />
+            </field>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_stock_available_kanban">
+        <field name="name">Quantity available to promise (kanban)</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.product_template_kanban_stock_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//span/field[@name='list_price']" position="after">
+                <field name="type" invisible="1" />
+                <field name="is_storable" invisible="1" />
+                <div
+                    t-if="record.show_on_hand_qty_status_button.raw_value"
+                >Available to Promise: <field name="immediately_usable_qty" /> <field
```

Comment:
Add the same group attribute as https://github.com/odoo/odoo/blob/686628f55f6ec/addons/stock/views/product_views.xml#L179 to align with core Odoo.